app/src/main/res/values-fr/strings.xml

```diff
@@ -45,10 +45,10 @@
     <string name="rename">Renommer</string>
   <string name="save">Sauvegarder</string>
     <string name="bookmarksadded">Favoris ajoutés</string>
-  <string name="fileexist">Un fichier avec le même nom existe déjà </string>
-    <string name="name">Nom: </string>
-  <string name="date">Date: </string>
-  <string name="location">Stocké dans: </string>
+  <string name="fileexist">Un fichier avec le même nom existe déjà</string>
+    <string name="name">Nom : </string>
```

Comment:
Why did you add the space here?

Reply:
In french punctuation a space is required before and after a colon.